### PR TITLE
IT-10213 Require php-coding-standard v1

### PR DESCRIPTION
**Pull request by IsaacStarkie**
Ticket: https://orchard.atlassian.net/browse/IT-10213

Updating the repository to use php-coding-standard v1

### CODE REVIEW

**Review by nemke — APPROVED** [review]

**Review by davorzdralo — APPROVED** [review]

**Comment by davorzdralo** [comment]
If you changed the composer.json, shouldn't composer.lock also change? At least the hash should be updated, right? @nemke am I wrong? Is v1.0 different from @dev at all?

**Comment by IsaacStarkie** [comment]
> 

It did change but composer.lock is in the .gitignore @davorzdralo

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^4.7",
-        "arbor-education/php-coding-standard": "@dev",
+        "arbor-education/php-coding-standard": "^1.0",
```
Comment (gecbla) [inline composer.json:21]: @IsaacStarkie  I think we are missing `composer.lock` file here also

**Review by gecbla — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (IsaacStarkie) [reply composer.json:21]: Please see my reply to Davor. I can add a brand new lock file but would have to remove it to .gitignore

**Review by gecbla — APPROVED** [review]

**Pull request merged**